src/parameter_search.ipynb

```diff
@@ -320,11 +320,11 @@
    "outputs": [],
    "source": [
     "searchable_index_params = {\n",
-    "    'chunk_size' : [192, 256, 320],\n",
-    "    'chunk_overlap' : [64, 128, 192],\n",
+    "    'chunk_size' : [128, 192, 256, 320],\n",
+    "    'chunk_overlap' : [32, 64, 96, 128, 160, 192],\n",
```

Comment:
is there a risk that this will run for a very long time? 
I think the best balance might be in providing few values with large intervals between them e.g. [32, 128, 192] in order to show the contrast in the final score with a relatively short time to complete. We want to minimize the time of wait + API costs for the end user

Reply:
agree